tests/test_utils.py

```diff
@@ -238,3 +240,37 @@ def test_get_data_from_napari_layer_squeeze(layer_data, selection):
     assert isinstance(result, np.ndarray)
     assert result.ndim == layer_data.ndim - 1
     assert np.array_equal(result, layer_data[selection].squeeze())
+
+
+# Dummy class to simulate BrainGlobeAtlas
+class DummyAtlas:
+    def __init__(self, reference, annotation):
+        self.reference = reference
+        self.annotation = annotation
+
+
+@pytest.fixture
+def dummy_atlas():
+    reference = np.array(
+        [[100, 150, 200], [50, 0, 75], [25, 25, 25]], dtype=np.uint16
+    )
+
+    annotation = np.array([[1, 0, 2], [0, 0, 3], [4, 0, 0]], dtype=np.uint16)
+
```

Comment:
```suggestion
    mock_atlas =mocker.patch(
        "brainglobe_registration.utils.utils.BrainGlobeAtlas"
    )
```

You can then add any attribute/function to this object and return it. See https://pytest-mock.readthedocs.io/en/latest/usage.html for more details.